pkg/apply/applier_test.go

```diff
@@ -875,7 +875,12 @@ func TestApplier(t *testing.T) {
 					ApplyEvent: &testutil.ExpApplyEvent{
 						GroupName:  "apply-0",
 						Identifier: testutil.ToIdentifier(t, resources["deployment"]),
-						Error:      testutil.EqualErrorType(inventory.NewInventoryOverlapError(fmt.Errorf(""))),
+						Operation:  event.Unchanged,
```

Comment:
I would like to change this to ApplySkipped in a future event operation refactor. But for now Unchanged == Skipped.

Reply:
Should I change it in this PR or a followup?

Reply:
Added ApplySkipped in https://github.com/kubernetes-sigs/cli-utils/pull/570